File: dbconnect.py
import mysql.connector

def get_db_connection():
    try:
        conn = mysql.connector.connect(
            host="localhost",
            user="root",
            password="@pokemon1",  # Change this to your actual password
            database="Variant"
        )
        if conn.is_connected():
            print("Database connected successfully!")
            return conn
    except mysql.connector.Error as e:
        print(f"Error connecting to MySQL: {e}")
        return None

File: dbconnect.php
<?php

$db_server = "localhost";
$db_user = "root";
$db_pass = "@pokemon1"; 
$db_name = "Variant";

// Attempt to connect to the database
$conn = mysqli_connect($db_server, $db_user, $db_pass, $db_name);

try{
    $conn = mysqli_connect($db_server, $db_user, $db_pass, $db_name);
}

catch(mysqli_sql_exception){
    echo"Connection Failed";
}

// Check the connection
if ($conn) {
    echo "";
}

?>
